src/styles/main.scss

```diff
@@ -0,0 +1,82 @@
+.watch {
+  height: 650px;
+  width: 650px;
+  border-radius: 50%;
+  border: 2px solid rgb(184, 207, 172);
+  background-color: rgb(253, 253, 67);
+  position: relative;
+  margin: 50px auto;
+  transition: 1s;
```

Comment:
It's better to set characteristics to which we apply a transition, to avoid problems.